docs/source/model_design/vivarium_overview/data_sources/index.rst

```diff
@@ -1,26 +1,157 @@
 .. _data:
 
-======================
-Understanding GBD Data
-======================
-
-.. note::
-   
-   Some of these links require access to the IHME VPN for access.
-   
-Currently, Vivarium utilizes GBD 2017 data. Understanding what data is available
-from this round and what modeling process produced it is a difficult task. The
-interface in Vivarium Inputs provides a function for pulling GBD 2017 data,
-but learning about it and understanding potential issues requires more effort.
-You may need to talk to GBD researchers, but the links below can help.
-
-- `GBD sources <http://pypi.services.ihme.washington.edu/docs/get_draws/latest-pre/sources.html>`_
-- `GBD trainings <https://hub.ihme.washington.edu/pages/viewpage.action?spaceKey=gbd2017&title=GBD+2017+Trainings>`_
-- `Epidemiological terminology <https://hub.ihme.washington.edu/download/attachments/44794562/Epi%20terms%20training.pptx?version=1&modificationDate=1512166429000&api=v2>`_
-
-Internal GBD teams also maintain up-to-date information scattered across a
-variety of R Shiny apps. A few are listed below. They require VPN access.
-
-- `GBD ID Tables <https://shiny.ihme.washington.edu/content/88/>`_
-- `The GBD Shared Hierarchy Editor <https://she.ihme.washington.edu/explore/location/version/446>`_
-- `The Risk Factor Toolbox <https://shiny.ihme.washington.edu/content/13/>`_
+===================================
+Understanding and Pulling GBD Data
+===================================
+
+Global Burden of Disease (GBD) Study data is a primary data source for our simulation models. 
```

Comment:
```suggestion
Global Burden of Disease (GBD) Study data is a fundamental data source for our simulation models. 
```

There is some term-of-art about "primary" vs "secondary" data and I thnk you just mean "very important"

Reply:
Yes, agreed, thanks for catching!